tests/Fixtures/Elasticsearch/Model/Tweet.php

```diff
@@ -73,12 +73,12 @@ public function setAuthor(User $author): void
         $this->author = $author;
     }
 
-    public function getDate(): ?\DateTimeInterface
```

Comment:
DateTimeInterface still works?

Reply:
Yes but you are right, I should add another test for `DateTimeInterface`.